en/reference/metrics.html

```diff
@@ -172,9 +179,72 @@ <h2 id="metrics-v1-values">/metrics/v1/values</h2>
   but does not report any metrics.
 </p>
 
+<h3 id="customizing-metric-sets">Customizing Metric Sets</h3>
+<p>
+    The aggregation metrics APIs will all return metrics from the
+    <a href="https://github.com/DataDog/integrations-extras/blob/master/vespa/metadata.csv">default</a> metric set.
+    While this should cover the most basic operational Vespa metrics, the set of metrics returned
+    can be customized through the <em><a href="services-admin.html#metrics">metrics</a></em> section in
+    <em><a href="services.html">services.xml</a></em>. You can control the metric set by either including a pre-defined
+    <em><a href="services-admin.html#metric-set">metric-set</a></em> or configure a specific
+    <em><a href="services-admin.html#metric">metric</a></em>. The latter way is also how you include any
+    <a href="#metrics-from-custom-components">metrics from custom components</a>.
+</p>
+
+<p>
+    Vespa comes with two pre-defined metric sets:
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <td>Metric set ID</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>default</td>
+            <td>
+                The default <a href="https://github.com/DataDog/integrations-extras/blob/master/vespa/metadata.csv">set of metrics</a>
+            </td>
+        </tr>
+        <tr>
+            <td>vespa</td>
+            <td>
+                A <a href="https://github.com/vespa-engine/vespa/blob/master/config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java">comprehensive list of metrics</a>
+                used to inspect low-level behavior of Vespa.<br/>
+                <b>NOTE:</b>The set of metrics here are not guaranteed to stay stable within a minor version of Vespa.
```

Comment:
We only remove metrics from this set on new major versions, no?

Reply:
yes

Reply:
Should we update text to remove note or leave it as-is?

Reply:
remove

Reply:
The note could instead warn about the potential cost of sending large number of metrics to external systems?